### ISSUE TITLE
Use Shortcodes Inside Custom Fields Created via JetEngine

### ISSUE THREAD

**Issue by coolsatinder** [CrocoBlock/suggestions#3684]
I create custom fields using JetEngine for woocommerce products.

These are text fields and I'm adding shortcodes inside them. Now I want to render these shortcodes via Elementor dynamic fields. Currently these custom fields are just printing the shortcode instead of rendering shortcode values. There must be a filter to render shortcodes from custom fields.

Like ACF provides a filter - `add_filter('acf/format_value/type=textarea', 'do_shortcode');`

https://support.advancedcustomfields.com/forums/topic/shortcodes-in-text-field/

**Comment by Lonsdale201** [comment]
With the Dynamic field widget, you can set, the shortcode option.

**Comment by coolsatinder** [comment]
Yes Dynamic Field has an option to do_shortcode via Filter Field Output but how to do this inside Elementor Pro dynamic fields?

Suppose I want to add a dynamic data inside Elementor Pro Icon Box then there is no option to do_shortcode.
![shortcodes-inside-custom-field](https://user-images.githubusercontent.com/26760823/126482035-344597da-bb54-4655-aa69-576bcee44906.jpg)
